tests/binary_tests/puppet.test.js

```diff
@@ -8,29 +9,31 @@ import { getDocument, queries, waitFor } from 'pptr-testing-library';
 
 jest.setTimeout(120000); // This test takes ~15 seconds, but longer in CI
 const PORT = 9009;
+const TMP_DIR = fs.mkdtempSync('tests/data/_');
```

Comment:
I noticed the switch in variable naming convention. Curious what made you go with all-cap underscores?

Reply:
Trying to make it clearer which variables have global scope.